commands/builds/create.js

```diff
@@ -81,9 +88,10 @@ module.exports = {
   help: 'Create build from contents of current dir',
   description: 'create build',
   flags: [
-    { name: 'source-url', description: 'Source URL that points to the tarball of your application\'s source code', hasValue: true },
-    { name: 'tar', description: 'Path to the executable GNU tar', hasValue: true },
-    { name: 'version', description: 'Description of your new build', hasValue: true }
+    { name: 'source-url', description: 'source URL that points to the tarball of your application\'s source code', hasValue: true },
+    { name: 'tar', description: 'path to the executable GNU tar', hasValue: true },
+    { name: 'version', description: 'description of your new build', hasValue: true },
+    { name: 'include-vcs-ignore', description: 'include files ignores by VCS (.gitignore, ...) from the build' }
```

Comment:
> maybe you could use the -X flag instead to read the .gitignore file? that one seems to be supported on all platforms

This would only work for customers using git, not mercurial or svn. While I'd expect it to be the vast majority of users, we weren't thinking anybody was using this CLI tool a few months ago. I'd quite cautious introducing this kind of breaking change.